mmdet/utils/deployment/symbolic.py

```diff
@@ -20,10 +20,37 @@
 from torch.onnx.symbolic_registry import register_op, get_registered_op, is_registered_op
 
 
-def py_symbolic(op_name=None, namespace='mmdet_custom'):
+def py_symbolic(op_name=None, namespace='mmdet_custom', adapter=None):
+    """
+    The py_symbolic decorator allows associating a function with a custom symbolic function
+    that defines its representation in a computational graph.
+
+    A symbolic function cannot receive a collection of tensors as arguments.
+    If your custom function takes a collection of tensors as arguments,
+    then you need to implement an argument converter from the collection 
+    and pass it to the decorator.
+
+    Args:
+        op_name (str): Operation name, must match the registered operation name.
+        namespace (str): Namespace for this operation.
+        adapter (function): Function for converting arguments.
```

Comment:
Please add a few words about how adapter should look like. It should have the same signature as the decorated function and should output `args` and `kwargs` for the symbolic call, right? Anything else?